cmd/docker-app/inspect.go

```diff
@@ -15,21 +16,47 @@ var (
 	inspectEnv            []string
 )
 
+// inspectCmd represents the inspect command
 func inspectCmd(dockerCli command.Cli) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "inspect [<app-name>] [-s key=value...] [-f parameters-file...]",
 		Short: "Shows metadata, parameters and a summary of the compose file for a given application",
 		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			app, err := packager.Extract(firstOrEmpty(args),
-				types.WithParametersFiles(inspectParametersFile...),
-			)
+			muteDockerCli(dockerCli)
+			appname := firstOrEmpty(args)
+			bundle, err := resolveBundle(dockerCli, "", appname)
 			if err != nil {
 				return err
 			}
-			defer app.Cleanup()
-			argParameters := cliopts.ConvertKVStringsToMap(inspectEnv)
-			return inspect.Inspect(dockerCli.Out(), app, argParameters)
+			params, err := parameters.LoadFiles(inspectParametersFile)
```

Comment:
It feels like the load / overrides / merge / stringKVToStringInterface thing might be extracted and reused between install/upgrade/inspect/status etc. cc @silvin-lubecki

Reply:
The refactoring is in progress, I'll update the inspect one in a followup.